src/main/resources/db/migration/all/20221110102259__add_status_column_to_premises_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE premises ADD COLUMN status TEXT;
```

Comment:
Since this isn't nullable in the Kotlin JPA model I think it'd be sensible to make it not-nullable in the DB schema too.  Probably sensible to set the Premises that are already in dev to active.